### PR TITLE
add new tests

### PR DESCRIPTION
**Pull request by AlekseiChapaev**
Added tests - checkJenkinsVersionTest, createPipelineWithCorrectNameTest, createPipelineWithEmptyNameTest and createPipelineWithDuplicateNameTest

### CODE REVIEW

**Review by NataliaKubiak — APPROVED** [review]

File: src/test/java/school/redrover/AlexChTest.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    public void checkJenkinsVersionTest(){
+        JenkinsUtils.login(getDriver());
+
+        Assert.assertEquals(getDriver().findElement(By.xpath("//div[@class = 'page-footer__links']/button"))
+                .getText().trim(), "Jenkins 2.414.2");
+    }
+
+    @Test
+    public void createPipelineWithCorrectNameTest(){
+        String jobName = "correctName";
```
Comment (Dmitry-Gonch) [inline src/test/java/school/redrover/AlexChTest.java:29]: Название теста пишем testCreatePipelineWithCorrect

File: src/test/java/school/redrover/AlexChTest.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    public void createNewPipeline(String pipelineName){
+        getDriver().findElement(By.xpath("//a[contains(@href, 'newJob')]")).click();
+        getDriver().findElement(By.id("name")).sendKeys(pipelineName);
+        getDriver().findElement(By.xpath("//li[contains(@class, 'FreeStyleProject')]")).click();
+        getDriver().findElement(By.id("ok-button")).click();
+        getDriver().findElement(By.name("Submit")).click();
+    }
+    
+    @Test
+    public void checkJenkinsVersionTest(){
```
Comment (Dmitry-Gonch) [inline src/test/java/school/redrover/AlexChTest.java:20]: Название теста testCheckJenkinsVersion

File: src/test/java/school/redrover/AlexChTest.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+        createNewPipeline(jobName);
+        getDriver().findElement(By.xpath("//a[contains(text(), 'Dashboard')]")).click();
+
+        Assert.assertTrue(getDriver().findElement(By.cssSelector("#projectstatus tbody tr"))
+                .getAttribute("id").contains("job_".concat(jobName)));
+    }
+
+    @Test
+    public void createPipelineWithEmptyNameTest(){
```
Comment (Dmitry-Gonch) [inline src/test/java/school/redrover/AlexChTest.java:40]: Также сначала test

File: src/test/java/school/redrover/AlexChTest.java (hunk 1 above)
Comment (AlekseiChapaev) [reply src/test/java/school/redrover/AlexChTest.java:29]: Thx. Tests were renamed

File: src/test/java/school/redrover/AlexChTest.java (hunk 2 above)
Comment (AlekseiChapaev) [reply src/test/java/school/redrover/AlexChTest.java]: Thx. Tests were renamed

File: src/test/java/school/redrover/AlexChTest.java (hunk 3 above)
Comment (AlekseiChapaev) [reply src/test/java/school/redrover/AlexChTest.java]: Thx. Tests were renamed

File: src/test/java/school/redrover/AlexChTest.java [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+        getDriver().findElement(By.xpath("//a[contains(@href, 'newJob')]")).click();
+        getDriver().findElement(By.xpath("//li[contains(@class, 'FreeStyleProject')]")).click();
+
+        Assert.assertEquals(getDriver().findElement(By.id("itemname-required")).getText(),
+                "» This field cannot be empty, please enter a valid name");
+    }
+
+    @Test
+    public void createPipelineWithDuplicateNameTest(){
```
[the remark this answers is not in the document]
Comment (AlekseiChapaev) [reply src/test/java/school/redrover/AlexChTest.java]: Thx. Tests were renamed

**Pull request merged**